backend/buildeR.Common/Enums/BuildStatus.cs

```diff
@@ -6,11 +6,11 @@ namespace buildeR.Common.Enums
 {
     public enum BuildStatus
     {
-        Error,
-        Failure,
-        Pending,
         Success,
+        Failure,
+        Error,
         Canceled,
-        InProgress
+        Pending,
+        InProgress,
```

Comment:
Was reordering necessary? It affects an API and DB that uses numbers

Reply:
I agree with Zhenia. You shouldn't change enum order. It can bring lots of issues. But I'll approve it for now since we don't have builds yet